gui/decision_view_widgets/decision_view_widget_create_holy_order.gui

```diff
@@ -0,0 +1,122 @@
+# Decision view widget to create a holy order
+# Required controller: create_holy_order
+
+flowcontainer = {
+	name = "decision_view_widget_create_holy_order"
+	direction = vertical
+	minimumsize = { 514 -1 }
+	maximumsize = { 514 -1 }
+	ignoreinvisible = yes
+	margin_top = 15
+
+	flowcontainer = {
+		visible = no # TODO [LS] TIT-6227 support custom names again
+
+		direction = horizontal
+		ignoreinvisible = yes
+		parentanchor = hcenter
+
+		text_single = {
+			parentanchor = vcenter
+			text = "CREATE_HOLY_ORDER_DECISION_NAME_LABEL"
+		}
+
+		widget = { size = { 15 0 } }
+
+		# When re-enabling, use editbox_standard instead.
+
+		#editbox = {
+		#	size = { 200 30 }
+		#	parentanchor = vcenter
+		#	alwaystransparent = no
+
+			# TODO [LS] TIT-6227 support custom names again
+			#text = "[DecisionViewWidgetCreateHolyOrder.GetCurrentName]"
+			#ontextedited = "[DecisionViewWidgetCreateHolyOrder.OnEditCurrentName]"
+		#}
+	}
+
+	widget = {
+		visible = no # TODO [LS] TIT-6227 support custom names again
+		size = { 0 5 }
+	}
+
+	flowcontainer = {
+		direction = horizontal
+		ignoreinvisible = yes
+		parentanchor = hcenter
+
+		text_single = {
+			visible = "[And(DecisionViewWidgetCreateHolyOrder.IsCreate, DecisionViewWidgetCreateHolyOrder.HasCurrentCapital)]"
+			parentanchor = vcenter
+			fontsize_min = 14
+			text = "CREATE_HOLY_ORDER_DECISION_CAPITAL_LABEL"
+		}
+
+		text_single = {
+			visible = "[And(DecisionViewWidgetCreateHolyOrder.IsRevoke, DecisionViewWidgetCreateHolyOrder.HasCurrentCapital)]"
+			parentanchor = vcenter
+			fontsize_min = 14
+			text = "REVOKE_HOLY_ORDER_DECISION_BARONY_LABEL"
+		}
+
+		spacer = {
+			size = { 5 5 }
+		}
+
+		text_single = {
+			visible = "[DecisionViewWidgetCreateHolyOrder.HasCurrentCapital]"
+			parentanchor = vcenter
+			fontsize_min = 14
+			default_format = "#high"
+			text = "[DecisionViewWidgetCreateHolyOrder.GetCurrentCapital.GetNameNoTier]"
+		}
+
+		widget = { size = { 15 0 } }
+
+		button_standard = {
+			name = "change_capital"
+			minimumsize = { 200 0 }
+			
+			onclick = "[DecisionViewWidgetCreateHolyOrder.OnSelectCapital]"
+			onclick = "[GetVariableSystem.Clear('move_portal')]"

```

Comment:
Do we need this file? I completely removed this overwrite and it seems to work fine.

This line is the only difference from the base game file, is it required for something?